README.md

```diff
@@ -191,6 +191,37 @@ This relies on a `locate` command to find repositories. This should be much fast
 
 *Note*: at this point, the plugin does not manage index update. Updating the index often requires to run a command like `updatedb` as root.
 
+#### MacOS
+
+`glocate` command used for caching on macos comes with gnu `findutils` which can be installed with
+```bash
+brew install findutils
+```
+With `glocate` installed use
```

Comment:
```suggestion
With `glocate` installed, add the following to your `.bashrc`/`.zshrc`
```
I suppose this needs to be added to some shell configuration, right? Which one did you use of `.bashrc` and `.zshrc`?